docs/OV_Runtime_UG/deployment/conditional_compilation_deployment.md

```diff
@@ -0,0 +1,289 @@
+# OpenVINO conditional compilation for optimal binary size{#opevino_conditional_compilation_deploying_guide}
+
+@sphinxdirective
+
+Conditional compilation can significantly reduce the binary size of OpenVINO™ package by excluding unnecessary components for inference of particular models, which makes the application benefit the optimal binary size.
+
+.. important::
+
+    The cost is OpenVINO runtime will only run inference for these particular models and the device for which was applied in conditional compilation.
```

Comment:
The tradeoff of conditional compilation is that the produced OpenVINO runtime can only run inference for the model(s) and platform(s) where conditional compilation was applied.